src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -118,24 +117,23 @@ export abstract class BaseEslintEngine extends AbstractRuleEngine {
 			const rules: Rule[] = [];
 
 			// Get all rules supported by eslint
-			const cli = this.baseDependencies.createCLIEngine(this.strategy.getCatalogConfig());
-			const allRules = this.strategy.filterDisallowedRules(cli.getRules());
+			const allRules = this.strategy.getRuleMap();
```

Comment:
We now get the rule map directly from the strategy instead of using the CLI engine.